trino/client.py

```diff
@@ -747,7 +748,11 @@ def execute(self, additional_http_headers=None) -> TrinoResult:
         if self.cancelled:
             raise exceptions.TrinoUserError("Query has been cancelled", self.query_id)
 
-        response = self._request.post(self._sql, additional_http_headers)
+        http_headers = {constants.HEADER_CLIENT_CAPABILITIES: 'PARAMETRIC_DATETIME'}
+        if additional_http_headers:
+            http_headers.update(additional_http_headers)
```

Comment:
this must go into the `http_headers` method instead where we already do the request header setting logic.

Reply:
Fixed